data/tests/sign.sh

```diff
@@ -0,0 +1,28 @@
+#/bin/bash
```

Comment:
If you've already got a script in place that can generate all these test keys and test certs, I would think it makes sense to just call that during test suite run (and .gitignore those files) rather than include particular cert .